src/register-service-worker.ts

```diff
@@ -0,0 +1,145 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on subsequent visits to a page, after all the
+// existing tabs open on the page have been closed, since previously cached
+// resources are updated in the background.
+
+// To learn more about the benefits of this model and instructions on how to
+// opt-in, read https://cra.link/PWA
+
+// [::1] is the IPv6 localhost address.
+// 127.0.0.0/8 are considered localhost for IPv4.
+const isLocalhost = Boolean(
```

Comment:
This file's code is pulled from the service worker code the CRA templates provide